README.md

```diff
@@ -100,11 +100,13 @@ The Exceptions Process is handled by the Release Team, please see their [documen
 
 | Label Name | Purpose | How to use this label | Who should use this label |
 | ------ | ------ | ------ | ------ |
-| `sig/foo` | Denotes the SIG(s) which owns this enhancement—e.g., `SIG Foo` | Set the label using the comment `/sig foo` (on a separate line) | Anyone |
-| `kind/feature` | Denotes that the issue should be tracked as an enhancement (all enhancement issues should be marked with this label) | Set the label using the comment `/kind feature` (on a separate line) | Anyone |
-| `tracked/yes` | Denotes an issue has been reviewed by a Feature Maintainer (SIG Release) and is actively tracked for the current milestone | Manually set | Feature Maintainers (SIG Release) ONLY |
-| `tracked/no` | Denotes an issue has been reviewed by a Feature Maintainer (SIG Release) and will not be actively tracked for the current milestone | Manually set | Feature Maintainers (SIG Release) ONLY |
+| `sig/foo` | Denotes the SIG(s) which owns the enhancement and other responsible SIG(s)-e.g., `SIG Foo` | Set the label using the comment `/sig foo` (on a separate line) | Anyone |
+| `kind/feature` | Denotes an issue should be tracked as an enhancement (all enhancement issues should be marked with this label) | Set the label using the comment `/kind feature` (on a separate line) | Anyone |
+| `tracked/yes` | Denotes an issue has been reviewed by the enhancements team (SIG Release) and is actively tracked for the current milestone | Set the label using the comment `/label tracked/yes` (on a separate line) | Release enhancements team ONLY |
+| `tracked/no` | Denotes an issue has been reviewed by the enhancements team (SIG Release) and will not be actively tracked for the current milestone | Set the label using the comment `/label tracked/no` (on a separate line) | Release enhancements team ONLY |
+| `tracked/out-of-tree` | Denotes an issue has an out-of-tree implementation | Set the label using the comment `/label tracked/out-of-tree` (on a separate line) | Release enhancements team ONLY |
```

Comment:
Would it be useful to elaborate what `out-of-tree implementation `means for new contributors?

Reply:
+1 possibly under https://github.com/kubernetes/enhancements/blob/master/docs/glossary.md (and maybe part of a separate PR?)

Reply:
I've added a bit more clarification on out-of-tree in here for now.